app.ts

```diff
@@ -801,11 +808,42 @@ app.get(
 	apisLimiter,
 	isAuthenticated,
 	async function (req, res) {
+		const sensorTypes = configManager.sensorTypes
+		const sensorGroups = configManager.namedScales
```

Comment:
I think the name is confusing - these are not sensor groups. These are named groups of scales.